### ISSUE TITLE
Framework7 is not a constructor

### ISSUE THREAD

**Issue by 509dave16** [framework7io/framework7#2452]
**TLDR** : It appears that framework7-react/utils/f7.js is the culprit for **Framework7 is not a constructor**

This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Framework7 Version: framework7-react@^3.0.0-beta.12 and/or framework7@3.0.0-beta.12
* Platform and Target: macOS High Sierra 10.13.5 , Chrome 67
* Live Link or JSFiddle: If what I provide below is not enough to go on let me know. Just wanted to at least get the conversation started regarding the bug I'm encountering with the latest beta release of framework7-react.

### What you did
- Attempted to follow the initialization steps outlined in your blog article about Phenome.js and Framework7 v3. 
- Tried following steps in a sample Preact Cli App and Create React App to make sure it wasn't a problem with React/Preact or the cli's bundling setup.
- There's one root App component and one TestPage component that has a route to it.
- Code snippet of sample app:
``` jsx
import react, {Component} from 'react';
import { render } from 'react-dom';
import {App as F7App, View, Page} from 'framework7-react';
const routes = [{path: '/',component: TestPage,}];

class TestPage extends Component {
    render() {
        return (
            <Page>
                <h1>Hello World</h1>
            </Page>
        );
    }
}

export default class App extends Component {
    render() {
        return (
            <F7App react={react} params={{theme: 'auto', routes, id: 'io.framework7.testapp'}}>
                <View url="/" main className="ios-edges"/>
            </F7App>
        )
    }

    componentDidMount() {
        this.$f7ready((f7) => {
            f7.dialog.alert('Component mounted');
        })
    }
}
```
**NOTE: Last to things below that I tried were just a shot in the dark. Probably not relevant**
- I also tried importing and instantiating an instance of Framework7 like in the steps at the following page: http://framework7.io/docs/installation.html
- Even found this issue with Framework7-Vue that looked related and tried adding a Webpack Alias for Framework7. But that didn't help either: https://github.com/framework7io/framework7-vue/issues/318
### Expected Behavior
- I expected for the app to load in the browser and display the TestPage component which has a simple message in it.
- I understand that I'm using a beta release so there could definitely be issues. Let me know if I should just be using the test app in the Phenome.js repo. Or the kitchen sink in the Framework7 repo.

### Actual Behavior
- The following errors were triggered
<img width="613" alt="f7-errors" src="https://user-images.githubusercontent.com/2925882/41814540-b0cf0dee-7704-11e8-9e80-de7130eb3e9e.png">
- Looking at the second error I took a look at the code location and it looks like the Framework7 object property that is being referenced is initialized as null and never changed. So it can't be a constructor and I don't see anywhere else in that location of it getting set to something other than null

<img width="1169" alt="source-of-f7-errors" src="https://user-images.githubusercontent.com/2925882/41814580-b121f142-7706-11e8-8142-5c169daac3b4.png">
